students/adam_wulw/lesson_01_python_basics/sum_of_digits.py

```diff
@@ -0,0 +1,4 @@
+num = int(input('Enter the number\n'))
+sum = (num % 10) + ((num % 100) // 10) + ((num % 1000) // 100)
```

Comment:
sum is built in function. So you redefined it
>>> sum([1,2,3])
6
>>> sum = 10
>>> sum([1,2,3])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'int' object is not callable

If I enter 111111 value it doesn't work